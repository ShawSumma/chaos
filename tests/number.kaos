num var5 = 5
var5 = 7
print var5
number var6 = 3.2
var6 = 5.4
print var6

num a = -123
num b = 0
b = b - 1
print a
print b
