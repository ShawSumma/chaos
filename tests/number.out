7
5.4
-123
-1
